src/slic3r/GUI/Widgets/WebView.cpp

```diff
@@ -218,7 +218,7 @@ wxWebView* WebView::CreateWebView(wxWindow * parent, wxString const & url)
 #ifndef __WIN32__
         });
 #endif
-        webView->EnableContextMenu(false);
+        webView->EnableContextMenu(true);
```

Comment:
I can revert this if we don't want it, but personally, I've had the page stuck on the mainsail "Initializing" view and couldn't get past that and couldn't figure out how to refresh the page. By enabling the context (i.e. right click) menu), I get a menu with a single item ("Reload") so that allows me to reload the page.

Reply:
Do you have a screen shot of the context menu?

Reply:
Sure! Here is what I see on macOS when I right click in the page. Just a small menu with the only option being "Reload". Though I'm not sure what it gives you on windows or linux.
<img width="670" alt="Screenshot 2023-05-28 at 9 50 49 AM" src="https://github.com/SoftFever/OrcaSlicer/assets/203336/77617646-3808-4a75-a20d-2c73242be0aa">